packages/tupaia-web/src/features/Map/MapOverlaySelector/MapTableModal.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { useParams } from 'react-router';
+import DownloadIcon from '@material-ui/icons/GetApp';
+import MuiIconButton from '@material-ui/core/IconButton';
+import { FlexColumn } from '@tupaia/ui-components';
+import { Typography } from '@material-ui/core';
+import { MapTable } from '@tupaia/ui-map-components';
+import { Modal } from '../../../components';
+import { useEntityAncestors, useMapOverlays } from '../../../api/queries';
+import { useMapOverlayData } from '../utils';
+
+interface entityObject {
```

Comment:
`interfaces` are usually capitalized, but I think in this case it might not be necessary, because we have an `Entity` type you can import